docs/CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -76,6 +82,22 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+# v1.9.10
+## Downloads for v1.9.10
+
+Download v1.9.10 in the [v1.9.10 release page](https://github.com/karmada-io/karmada/releases/tag/v1.9.10).
+
+## Changelog since v1.9.9
+### Changes by Kind
+#### Bug Fixes
+- `karmadactl`: Fixed `karmada-metrics-adapter` use the incorrect certificate issue when deployed via karmadactl `init`. ([#5859](https://github.com/karmada-io/karmada/pull/5859), @seanlaii)
+- `karmada-controller-manager`: Fixed the corner case where the reconciliation of aggregating status might be missed in case of component restart. ([#5884](https://github.com/karmada-io/karmada/pull/5884), @liangyuanpeng)
+- `karmada-scheduler`: Avoid filtering out clusters if the API enablement is incomplete during re-scheduling. ([#5932](https://github.com/karmada-io/karmada/pull/5932), @XiShanYongYe-Chang)
+
+#### Others
+- The feature `Failover` now has been disabled by default, which should be explicitly enabled to avoid unexpected incidents. ([#5948](https://github.com/karmada-io/karmada/pull/5948), @XiShanYongYe-Chang)
+- The base image `alpine` now has been promoted from `alpine:3.20.3` to `alpine:3.21.0`. ([#5922](https://github.com/karmada-io/karmada/pull/5922))
```

Comment:
```suggestion
#### Urgent Upgrade Notes
- The feature `Failover` now has been disabled by default, which should be explicitly enabled to avoid unexpected incidents. ([#5948](https://github.com/karmada-io/karmada/pull/5948), @XiShanYongYe-Chang)

If you are using the feature `Failover`, please enable it explicitly by adding the `--feature-gates=Failover=true,<other feature>`
flag to the `karmada-controller-manager` component. If you are not using this feature, this change will have no impact.

#### Bug Fixes
- `karmadactl`: Fixed `karmada-metrics-adapter` use the incorrect certificate issue when deployed via karmadactl `init`. ([#5859](https://github.com/karmada-io/karmada/pull/5859), @seanlaii)
- `karmada-controller-manager`: Fixed the corner case where the reconciliation of aggregating status might be missed in case of component restart. ([#5884](https://github.com/karmada-io/karmada/pull/5884), @liangyuanpeng)
- `karmada-scheduler`: Avoid filtering out clusters if the API enablement is incomplete during re-scheduling. ([#5932](https://github.com/karmada-io/karmada/pull/5932), @XiShanYongYe-Chang)

#### Others
- The base image `alpine` now has been promoted from `alpine:3.20.3` to `alpine:3.21.0`. ([#5922](https://github.com/karmada-io/karmada/pull/5922))

```